.github/workflows/osrm-backend.yml

```diff
@@ -449,53 +463,6 @@ jobs:
             ENABLE_CONAN: ON
             NODE_PACKAGE_TESTS_ONLY: ON
 
-          - name: conan-macos-x64-release-node-lts
```

Comment:
LTS is Node 18 at the moment, i.e. it seems there is no reason to build it separately as we explicitly build Node 18 version now.

Reply:
Just for better understanding: node-latest is Node 19 now.